src/components/Comments/ContainerComment.js

```diff
@@ -247,9 +247,10 @@ const ContainerComment = ({
           </View>
           <View style={{paddingLeft: 22}}>
             <View style={[styles.containerComment]}>
+              <View style={styles.lastConnectorBlocker} />
               <View style={styles.lineBeforeProfile} />
               {comments.map((item, index) => (
-                <View key={`p${index}`}>
+                <View key={`p${index}`} style={{zIndex: 10}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `View` component with the style `lastConnectorBlocker` is added here. However, it's not clear from the code how this component behaves when there are no comments. You should consider handling the case where the `comments` array is empty to prevent any potential visual issues.
```diff
- <View style={styles.lastConnectorBlocker} />
+ {comments.length > 0 && <View style={styles.lastConnectorBlocker} />}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->